src/main/java/reposense/parser/CsvParser.java

```diff
@@ -54,8 +54,9 @@ public CsvParser(Path csvFilePath) throws IOException {
         List<T> results = new ArrayList<>();
         Iterable<CSVRecord> records;
 
-        try (Reader csvReader = new FileReader(csvFilePath.toFile())) {
-            records = CSVFormat.DEFAULT.withFirstRecordAsHeader().parse(csvReader);
+        try (BufferedReader csvReader = new BufferedReader(new FileReader(csvFilePath.toFile()))) {
```

Comment:
Prefer `try (Reader csvReader ...`

Reply:
Thanks for the review @yong24s.
I think we'll need to use `BufferedReader` here, since the `getHeader` method relies on the `BufferedReader`'s ability to read lines (as documented [here](https://docs.oracle.com/javase/8/docs/api/java/io/BufferedReader.html#readLine--))